thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/DetectionResourceHttpUtils.java

```diff
@@ -43,4 +45,23 @@ public String runBackfillAnomalyFunction(String id, String startTimeIso, String
         DETECTION_JOB_ENDPOINT + id + BACKFILL + "?start=" + startTimeIso + "&end=" + endTimeIso + "&force=" + forceBackfill);
     return callJobEndpoint(req);
   }
+
+  public String runAutoTune(String function_id, String startTimeIso, String endTimeIso, String autoTuneType) throws Exception {
+    HttpPost req = new HttpPost(
+        DETECTION_JOB_ENDPOINT + AUTOTUNE_FILTER + function_id
+            + "?startTime=" + startTimeIso
+        + "&endTime=" + endTimeIso
+        + "&autoTuneType=" + autoTuneType
+    );
+    return callJobEndpoint(req);
+  }
+
+  public String getEvalStatsAlertFilter(String function_id, String startTimeIso, String endTimeIso) throws Exception{
```

Comment:
follow one convention for time, use long as you would not have to worry about TZ.